packages/next-auth/src/index.tsx

```diff
@@ -384,11 +400,15 @@ export default function NextAuth(
         setEnvDefaults(_config)
         return signOut(options, _config)
       },
-      unstable_update: (data) => {
+      update: (data) => {
         const _config = config(undefined)
         setEnvDefaults(_config)
         return update(data, _config)
       },
+      unstable_auth: initFutureAuth(config, setEnvDefaults),
+      unstable_update: () => {
+        throw new TypeError("Not yet implemented.")
```

Comment:
Quick note, I'd make a mention here of the rename of the previous `unstable_update` to `update`. Just for anyone whos already potentially using it. If they start seeing `"Not yet implemented"` errors, they'll be very confused.

Maybe something like, `The previous implementation has been renamed 'update', this method is now not implemented` :thinking: